packages/neovim/build.sh

```diff
@@ -22,16 +22,38 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 "
 TERMUX_PKG_CONFFILES="share/nvim/sysinit.vim"
 
+_patch_luv() {
+	# git submodule update --init deps/lua-compat-5.3 failed
+	cp -r $1/build/src/lua-compat-5.3/* $1/build/src/luv/deps/lua-compat-5.3/
+	cp -r $1/build/src/luajit/* $1/build/src/luv/deps/luajit/
+	cp -r $1/build/src/libuv/* $1/build/src/luv/deps/libuv/
+}
+
 termux_step_host_build() {
 	termux_setup_cmake
 
+	TERMUX_ORIGINAL_CMAKE=$(which cmake)
+	if [ ! -f "$TERMUX_ORIGINAL_CMAKE.orig" ]; then
+		mv "$TERMUX_ORIGINAL_CMAKE" "$TERMUX_ORIGINAL_CMAKE.orig"
+		cp "$TERMUX_PKG_BUILDER_DIR/custom-bin/cmake" "$TERMUX_ORIGINAL_CMAKE"
+		chmod +x "$TERMUX_ORIGINAL_CMAKE"
+	fi
+	export TERMUX_ORIGINAL_CMAKE="$TERMUX_ORIGINAL_CMAKE.orig"
+
 	mkdir -p $TERMUX_PKG_HOSTBUILD_DIR/deps
 	cd $TERMUX_PKG_HOSTBUILD_DIR/deps
 	cmake $TERMUX_PKG_SRCDIR/third-party
-	make -j 1
+
+	make -j 1 \
+	|| (_patch_luv $TERMUX_PKG_HOSTBUILD_DIR/deps && make -j 1 ) \
+	|| bash
```

Comment:
Why is bash run on error?

Reply:
Ah i use that to debug when it failed, forgot to remove